mantle/kola/tests/ignition/security.go

```diff
@@ -61,9 +61,10 @@ func init() {
 			"TLSServe": register.CreateNativeFuncWrap(TLSServe),
 		},
 		Tags: []string{"ignition"},
-		// DO: https://github.com/coreos/bugs/issues/2205
-		// Packet & QEMU: https://github.com/coreos/ignition/issues/645
-		ExcludePlatforms: []string{"do", "packet", "qemu"},
+		// QEMU: When we moved to unprivileged QEMU we lost support for multiple QEMU VMs 
+		// communicating on a private network. 
```

Comment:
Nit: I'm not a fan of comments saying what we've done in the past; today's code should stand on its own.  We could just say "qemu-unpriv doesn't support multiple VMs communicating with each other", and drop the link to the removal commit.

Reply:
works for me

Reply:
fixed in latest push